apps/workflowengine/lib/Settings/ASettings.php

```diff
@@ -108,6 +117,12 @@ public function getForm() {
 			$this->getScope()
 		);
 
+		$this->initialStateService->provideInitialState(
+			Application::APP_ID,
+			'appstoreenabled',
+			$this->config->getSystemValueBool('appstoreenabled')
```

Comment:
```suggestion
			$this->config->getSystemValueBool('appstoreenabled', true)
```

to  honour the default. Otherwise the card won't be shown despite the appstore being enabled.

Reply:
Good point 👍